inst/sql/sql_server/ComputeTimeSeries.sql

```diff
@@ -34,6 +35,54 @@ INNER JOIN #calendar_periods cp ON (
 		AND cohort_end_date <= period_end
 		)
 WHERE cohort_definition_id IN (@cohort_ids)
-GROUP BY period_begin
-	,calendar_interval
-	,cohort_definition_id;
+GROUP BY period_begin,
+	calendar_interval,
+	cohort_definition_id
+
+UNION
+
+SELECT cohort_definition_id cohort_id,
+	period_begin,
+	calendar_interval,
+	'o' prevalence_type,
+	COUNT_BIG(*) records,
+	COUNT_BIG(DISTINCT person_id) subjects,
+	SUM(datediff(dd, CASE 
+				WHEN observation_period_start_date >= period_begin
+					THEN observation_period_start_date
+				ELSE period_begin
+				END, CASE 
+				WHEN observation_period_end_date >= period_end
+					THEN period_end
+				ELSE observation_period_end_date
+				END) + 1) person_days,
+	COUNT_BIG(CASE 
+			WHEN observation_period_start_date >= period_begin
+				AND observation_period_start_date <= period_end
+				THEN subject_id
+			ELSE NULL
+			END) records_incidence,
+	COUNT_BIG(DISTINCT CASE 
+			WHEN observation_period_start_date >= period_begin
+				AND observation_period_start_date <= period_end
+				THEN subject_id
+			ELSE NULL
+			END) subjects_incidence
+FROM @cdm_database_schema.observation_period o
```

Comment:
When running on an existing project this line throws an error for me, but I'm re-running to confirm it wasn't an error in my R session.